packages/roosterjs-content-model-api/lib/publicApi/table/applyTableBorderFormat.ts

```diff
@@ -366,6 +369,12 @@ export function applyTableBorderFormat(
                     modifyPerimeter(tableModel, sel, borderFormat, perimeter, isRtl);
                 }
 
+                const tableMeta = hasMetadata(tableModel) ? updateTableMetadata(tableModel) : {};
```

Comment:
From my PR https://github.com/microsoft/roosterjs/pull/2641, I just checked in a new API `getTableMetadata`.

Can you use this new API instead?